README.md

```diff
@@ -41,11 +41,12 @@ See the [documentation](https://soundata.readthedocs.io/) for more examples and
 
 ### Currently supported datasets
 
-* UrbanSound8K
 * ESC-50
+* URBAN-SED
+* UrbanSound8K
 * More added soon!
```

Comment:
I'm good with having this for now but it will become quickly intractable as we add datasets, in which case we should only keep the link below